### PR TITLE
Added Urban Dictionary command

### PR DESCRIPTION
**Pull request by decavacado**
Added Urban Dictionary command came accross this when scrolling through github for projects to contribute too

### CODE REVIEW

File: lib/commands/fun/urban.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+var { MessageEmbed } = require("discord.js")
+var embed = MessageEmbed
+var fetch = require('node-fetch')
+
+class Chuck extends commandos.Command {
+    constructor(client){
+        super(client, {
+            name: "urban",
+            aliases: ["dict","simp"],
+            group: "fun",
```
Comment (Syntthetix) [inline lib/commands/fun/urban.js:11]: `group: "misc"` to match current bot config

File: lib/commands/fun/urban.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                key: "keyword",
+                prompt: "What do you want to search up?",
+                type: "string"
+            }]
+        })
+    }
+    async run(msg, {keyword}) {
+        let send_em = new embed()
+        send_em.setTitle("Urban")
+        send_em.setColor("#7900a8")
```
Comment (Syntthetix) [inline lib/commands/fun/urban.js:26]: `setColor(0x7900a8) to match format of other embed code

File: lib/commands/fun/urban.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        send_em.setFooter("Powered by Urban dictionary")
+        let res = await fetch(`https://mashape-community-urban-dictionary.p.rapidapi.com/define?term=${keyword}`, {headers: {"x-rapidapi-host": "mashape-community-urban-dictionary.p.rapidapi.com", "x-rapidapi-key": "50d3bf2303msha7f835672a0a9acp13af12jsn0636fc6246bb"}})
```
Comment (Syntthetix) [inline lib/commands/fun/urban.js:28]: Would want to remove the API key, move it to environment variables, then pass variable as parameter here. All that best practices stuff. `"x-rapidapi-key": process.env.X-RAPIDAPI-KEY` <-- Just use that and I can add env variables on my end.

File: lib/index.js [hunk 4]
```diff
@@ -25,7 +25,8 @@ client.registry
   .registerGroups([
     ['misc', 'Misc'],
     ['util', 'Util'],
-    ['game', 'Game']
+    ['game', 'Game'],
+    ['fun', 'Fun']
```
Comment (Syntthetix) [inline lib/index.js:29]: I would just remove the fun group and put this command under misc instead.

**Comment by decavacado** [comment]
The right commit is Changes2

**Comment by Syntthetix** [comment]
Looks good to me, thanks a lot :)

**Comment by decavacado** [comment]
Np

**Pull request merged**